### DISCUSSION TITLE
Add support for Django Signals

### DISCUSSION THREAD

**Discussion by ZuluPro** [Archmonger/django-dbbackup#578] [Feature Requests]
[Django Signals](https://docs.djangoproject.com/en/5.2/topics/signals/) could be a good way to let developers interact with DBBackup's actions. I imagine the following:
- pre_dbbackup & post_dbbackup
- pre_dbrestore & post_dbrestore
- pre_mediabackup & post_mediabackup
- pre_mediarestore & post_mediarestore

Would help users make a lock or flush something before backups.

**Comment by Archmonger** [comment]
This feature has been added within #614
